dist/css/uswds.css

```diff
@@ -1381,15 +1381,16 @@ textarea {
 }
 
 select {
+  appearance: none;
```

Comment:
I think it makes more sense to keep this as it was before, since the vendor-prefixed properties begin with a dash and it keeps the ones that begin with letters together. This is what thoughtbot Bourbon does: https://github.com/thoughtbot/bourbon/blob/master/core/bourbon/library/_prefixer.scss#L19